tests/ft_rust.vader

```diff
@@ -53,3 +53,32 @@ Execute (rust: cargo: uses directory with Cargo.toml as cwd):
   let maker = neomake#makers#ft#rust#cargo()
   AssertEqual maker.cwd, tempdir
   bwipe
+
+Execute (cargo test failures):
+  new
+  file build/lib.rs
+  let cargotest = NeomakeTestsGetMakerWithOutput('neomake#makers#ft#rust#cargotest',
+    \ '../test/fixtures/rust/cargotest_with_failures')
+
+  call neomake#Make(1, [cargotest])
+  NeomakeTestsWaitForFinishedJobs
+
+  AssertEqual 1, len(getloclist(0))
```

Comment:
Please use something like `AssertEqual map(getloclist(0), 'v:val.text'), ['error message']` to assert some property at least.

Reply:
Ok, will read a bit more docs for that.